src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesSpec.scala

```diff
@@ -198,4 +198,21 @@ class UserRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with
       status shouldEqual StatusCodes.Forbidden
     }
   }
+
+  "DELETE /admin/user/{userSubjectId}/petServiceAccount/{project}" should "delete a pet (as an admin)" in withAdminRoutes { (samRoutes, adminRoutes) =>
+    Post("/register/user") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Created
+      responseAs[UserStatus] shouldEqual UserStatus(UserStatusDetails(defaultUserId, defaultUserEmail), Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true))
+    }
+
+    Delete(s"/api/admin/user/$defaultUserId/petServiceAccount/myproject") ~> adminRoutes.route ~> check {
+      status shouldEqual StatusCodes.NoContent
```

Comment:
Are there service-level specs that test the per-project `deleteUserPetServiceAccount` method?

IIRC the API specs use a mock LDAP but the service specs use a real LDAP, so the actual delete functionality could be tested in there.

Reply:
It's exercised in `GoogleEntensionSpec`

Reply:
cool